### PR TITLE
Update master.sh

### PR DESCRIPTION
**Pull request by mehul-lb**
Some updates as per https://kubernetes.io/blog/2023/08/15/pkgs-k8s-io-introduction/#how-to-migrate

### CODE REVIEW

File: Ubuntu/master.sh [hunk 1]
```diff
@@ -137,6 +137,8 @@ function serviceStatusCheck() {
 }
 
 echo "2. Install kubeadm, kubectl and kubelet:"
+# Doc https://kubernetes.io/blog/2023/08/15/pkgs-k8s-io-introduction/#how-to-migrate
+mkdir mkdir /etc/apt/keyrings
```
Comment (ajay-lb) [inline Ubuntu/master.sh:141]:
```suggestion
mkdir /etc/apt/keyrings
```

**Review by mehul-lb — COMMENTED** [review]
Updated

**Review by ajay-lb — APPROVED** [review]

File: Ubuntu/master.sh [hunk 2]
```diff
@@ -137,6 +137,8 @@ function serviceStatusCheck() {
 }
 
 echo "2. Install kubeadm, kubectl and kubelet:"
+# Doc https://kubernetes.io/blog/2023/08/15/pkgs-k8s-io-introduction/#how-to-migrate
```
Comment (aditya-lb) [inline Ubuntu/master.sh:140]:
```suggestion
# For versions older than 22.04, the /etc/apt/keyrings directory does not exist by default so it needs to be created. See https://kubernetes.io/blog/2023/08/15/pkgs-k8s-io-introduction/#how-to-migrate for more details.
```

File: Ubuntu/master.sh [hunk 3]
```diff
@@ -137,6 +137,8 @@ function serviceStatusCheck() {
...
+mkdir /etc/apt/keyrings
```
Comment (aditya-lb) [inline Ubuntu/master.sh:141]:
```suggestion
mkdir -p /etc/apt/keyrings
```

**Review by aditya-lb — CHANGES_REQUESTED** [review]

**Pull request closed without merging**